src/lib/zip-format.js

```diff
@@ -57,32 +56,12 @@ var commonZipRanges = [
 
 commonZipRanges.sort((a, b) => (a[0] - b[0]))
 
-export const zipToTimeZone = function (zip, r) {
+export const zipToTimeZone = (zip) => {
   // will search common zip ranges -- won't necessarily find something
   // so fallback on looking it up in db
-  if (typeof zip == 'number' || zip.length >= 5) {
-    zip = parseInt(zip)
-    return commonZipRanges.find((g) => (zip >= g[0] && zip < g[1]))
+  if (typeof zip === 'number' || zip.length >= 5) {
+    const parsed = parseInt(zip, 10)
+    return commonZipRanges.find((g) => (parsed >= g[0] && parsed < g[1]))
   }
-}
-
-export const findZipRanges = function (r) {
```

Comment:
This isn't used in code, but it's how to generate the data higher up in `commonZipRanges` so let's not delete, please.

Reply:
I'm not sure this function works as intended--the final `return` is unreachable, so `zipchanges` gets populated and then discarded. I'm restoring this function but commenting it out--we can uncomment and fix it separately.